physical/raft/raft_test.go

```diff
@@ -203,6 +204,61 @@ func TestRaft_Backend(t *testing.T) {
 	physical.ExerciseBackend(t, b)
 }
 
+func TestRaft_Backend_LargeValue(t *testing.T) {
+	b, dir := getRaft(t, true, true)
+	defer os.RemoveAll(dir)
+
+	t.Helper()
+
+	value := make([]byte, defaultKVMaxValueSize+1)
+	rand.Read(value)
+	entry := &physical.Entry{Key: "foo", Value: value}
+
+	if err := b.Put(context.Background(), entry); err == nil {
+		t.Fatal("expected error for put entry")
+	}
```

Comment:
Do we want to include a string check on the error to make sure that's the error that we want after this nil check?
```
if !strings.Contains(err.Error(), physical.ErrValueTooLarge) {
	t.Fatalf("expected %q, got %v", physical.ErrValueTooLarge, err)
}
```